swagger/swagger.yaml

```diff
@@ -1660,7 +1658,7 @@ components:
         - treated
     AvpuLevelEnum:
       type: string
-      description: Character level of response.  See [Levels of Response](https://www.firstresponse.org.uk/first-aid-az/3-general/first-aid/79-levels-of-response) for details
+      description: Character level of response; anything but ALERT is considered unconscious.  See [Levels of Response](https://www.firstresponse.org.uk/first-aid-az/3-general/first-aid/79-levels-of-response) for details
```

Comment:
This is weird. How can pain and voice be unconscious? The link seems to agree that voice and pain have some level of consciousness...

Reply:
A performer linked [this article](https://www.ems1.com/ems-training/articles/use-avpu-scale-to-determine-a-patients-level-of-consciousness-FVpjgzNGwSJAGoeQ/).  AVPU measures "levels of consciousness," but to be considered "conscious," breathing and talking/vocalizing is not sufficient.  See the section, "Being Not Awake is unconscious" especially.

Reply:
I only cared because I was thinking it would affect the sim, but now I'm thinking it probably doesn't. It's still weird to me, but okay!